ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -107,6 +109,15 @@ oc wait clusteroperators/machine-config --for=condition=Upgradeable=true --timeo
 EOF
 
       TEST_UPGRADE_ARGS="--from-repository ${DS_REGISTRY}/localimages/local-test-image"
+      TEST_SKIPS="\[sig-arch\]\[Early\] APIs for openshift\.io must have stable versions \[Suite:openshift/conformance/parallel\]"
```

Comment:
Could we set this in the job definition? Similare to here
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/installer/openshift-installer-release-4.17.yaml#L604

Reply:
I added it to the test chain that will use the baremetal-e2e-test script